src/components/Drawer/styles.js

```diff
@@ -5,6 +5,10 @@ export const IconContainer = styled.div`
     svg{
       font-size: ${({theme})=>theme.scale.h5};
     }
+
```

Comment:
try using the `isMobile` value from the useMediaQuery hook on the parent element instead of hardwiring the mediaQuery everywhere else